<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editContactDialog</class>
 <widget class="QDialog" name="editContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit the contact</string>
  </property>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>30</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Book</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contact Name</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>531</width>
     <height>141</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaNumbers">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>139</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>451</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="font">
       <font>
        <family>Franklin Gothic Book</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete contact</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OKButon">
      <property name="font">
       <font>
        <family>Franklin Gothic Book</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addNumberButton">
      <property name="font">
       <font>
        <family>Franklin Gothic Book</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add number</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
